rkale/operations.py

```diff
@@ -15,7 +15,7 @@ def get_input(verbose_info):
 def get_answers(operation, checks, paths, force=False):
     answers = []
     for check, (source, destination) in zip(checks, paths):
-        missing_src, missing_dst = read(check.src_out), read(check.dst_out)
+        missing_src, missing_dst, diff = read(check.src_out), read(check.dst_out), read(check.diff_out)
```

Comment:
minor: I personally would keep these as separate statements on separate lines for readability :)